cnf-certification-test/manageability/suite.go

```diff
@@ -73,6 +73,11 @@ func containerPortNameFormatCheck(portName string) bool {
 }
 
 func testContainerPortNameFormat(env *provider.TestEnvironment) {
+	if len(env.ProtocolName) > 0 {
```

Comment:
There is no need for the `if`.

Reply:
you'r right but is still works, anyway I will change and push again.